Hacktoberfest-2022 Update blog page

Anuran12 opened the pull request:
## Related Issue

- Information about the related issue
Close: #196 

### Describe the changes you've made

I have change the Blog page... I made this change according to the contributors page theme 

## Type of change

What sort of change have you made:
<!--
Example how to mark a checkbox:-
- [x] My code follows the code style of this project.
-->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update.
- [ ] This change requires a documentation update

## How Has This Been Tested?
I created a local hosting to test the changes . I tested for various screen ratio to check the responsive

## Checklist
<!--
Example how to mark a checkbox:-
- [x] My code follows the code style of this project.
-->
- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

 Original           | Updated
 :--------------------: |:--------------------:
 original screenshot | ![image](https://user-images.githubusercontent.com/104062485/193422383-33051333-ec70-4fb9-b3f6-000973103170.png)

## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/WebXDAO/.github/blob/main/CODE_OF_CONDUCT.md)


<a href="https://gitpod.io/#https://github.com/WebXDAO/WebXDAO.github.io/pull/210"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>

krshkodes reviewed (changes requested):
Can you let the breakpoints and variable names remain the same? They might cause issues elsewhere otherwise

Anuran12 commented:
> Can you let the breakpoints and variable names remain the same? They might cause issues elsewhere otherwise

well the breakpoints are already using by default .
![image](https://user-images.githubusercontent.com/104062485/193436916-52237855-969d-4347-8fb4-82f91ee93d9c.png)

the sm,md,lg.xl are default screen size for tailwind.. 

![image](https://user-images.githubusercontent.com/104062485/193436966-9256c3c2-8fa2-4fee-bd04-e0278ad506b5.png)

And those are already using in "Navbar.js" 

I just added the 'xs' additionally . because tailwind does not provide us 360px screen width by default .

and without that the blog cards are looking bad in phone view. so i just added 'xs': '360px',

I guess this will not harm any other code.

krshkodes reviewed (changes requested):
Tailwind does provide xs, but it isn't the size you want - I would suggest that you use a grid layout for responsiveness and not touch the tailwind config more than needed

Anuran12 commented:
> Tailwind does provide xs, but it isn't the size you want - I would suggest that you use a grid layout for responsiveness and not touch the tailwind config more than needed

@krshkodes I Made changes according to your advice.. Please check it : )

mkubdev commented on src/pages/blog.js:
```diff
@@ -1,34 +1,69 @@
+  };
+};
+
+const blogs = ({ data }) => {
```
```suggestion
const Blog = ({ data }) => {
```


Hi, [Pages]https://nextjs.org/docs/basic-features/pages) should be exported with Capitalized letter! :D

mkubdev commented on src/pages/blog.js:
```diff
@@ -1,34 +1,69 @@
-    revalidate: 10,
-  };
-}
+export default blogs;
```
```suggestion
export default Blog;
```

mkubdev commented on src/pages/blog.js:
```diff
@@ -1,34 +1,69 @@
+          <h1 className='w-full my-2 text-4xl font-bold leading-tight text-center text-white'>
+            Blogs
+          </h1>
+          <div className='w-full mb-4'>
```
```suggestion
          <div className="w-full mb-4">
```

Hi please, kindly suggest you to update the className with double quote: ` " `

mkubdev reviewed (changes requested):
Hi 🎉
This look good to me! But it requires few changes.

If you use VSCode, i suggest you to use the [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) plugin to format your files. So that single quotes are auto formated into double quotes. 🥳

Good job ⚡🎉

Anuran12 commented:
@mkubdev Done...

mkubdev reviewed (approved):
This look good to me!

vinzvinci reviewed (approved):
LGTM!

Anuran12 commented:
@mkubdev you can marge this.. because this page is no need any update or improvement now.

mkubdev commented:
Hello @Anuran12 !

Yes, let's merge your changes. Thanks a lot for your valuable contribution and your implication. Very good work! ⚡

Please join us on our [Discord](https://discord.gg/fah5cTBN), we are looking for maintainers/collaborators!

Anuran12 commented:
Hey.. @mkubdev 

I have joined Discord..

I would like to be a maintainers/collaborators.. But I am a newbie so please guide me. This opportunity will give me more exposer.

Thanks  : )

mkubdev commented:
What is your Discord username?
Cool if you already joined. Don't worry, we all started learning like you!

Anuran12 commented:
Anuran#2573
